app/src/panels/time-series/time-series.vue

```diff
@@ -1,17 +1,20 @@
 <template>
-	<div ref="chartEl" class="time-series" />
+	<div class="time-series">
+		<v-progress-circular v-if="loading" indeterminate />
```

Comment:
Does every panel render this on loading? We might want to make a centralized loading state in dashboard.vue instead. That way we can do an easier

```html
<v-progress-circular v-if="loading" indeterminate />
<component v-else :is="`panel-${panel.id}`" etc />
```